gym/wrappers/monitoring/video_recorder.py

```diff
@@ -10,12 +10,16 @@
 import numpy as np
 
 from gym import error, logger
+from gym.utils import closer
 
 
 def touch(path):
     open(path, "a").close()
 
 
+video_recorder_closer = closer.Closer()
```

Comment:
I believe using Closer() is not actually necessary at all for the new test to pass - as it is garbage collector that calls `__del__` directly, which then calls close(). My understanding is Closer() is only useful in the cases when you explicitly call Closer().close() before terminating the program.

Reply:
I don't think we need the closer either, as we have called `close()` in `__del__()`. I add this closer here to follow the design of `Monitor`:

https://github.com/openai/gym/blob/8da6224b7263f9541a55d541d8268f68ece7d99e/gym/wrappers/monitor.py#L178-L196

And as commented at the top of this PR (https://github.com/openai/gym/pull/2318#issue-713169148):

> gym.wrappers.Monitor has been registered with a closer and can be autoclosed at the program exit. However, for low level API users, VideoRecorder itself may should have autoclose feature as well.